src/components/FloatingActionButton.tsx

```diff
@@ -88,6 +92,18 @@ function FloatingActionButton({onPress, isActive, accessibilityLabel, role, isTo
         onPress(event);
     };
 
+    const longPressFabAction = (event: GestureResponderEvent | KeyboardEvent | undefined) => {
+        // Only execute on narrow layout - prevent event from firing on wide screens
+        if (!shouldUseNarrowLayout) {
+            return;
+        }
+
+        hideProductTrainingTooltip();
+        // Drop focus to avoid blue focus ring.
```

Comment:
```suggestion

        // Drop focus to avoid blue focus ring.
```
ESLint should have caught that there should be an empty line before comments. Hm 🤔